File: pinyougou-web-shop/src/main/webapp/js/service/ordersService.js
//服务层
app.service('ordersService',function($http){
	    	
	//读取列表数据绑定到表单中
    this.findAll=function(){
		return $http.get('../orders/findAll.do');
	}
    //搜索
    this.search=function(page,rows,searchEntity){
        return $http.post('../orders/search.do?page='+page+"&rows="+rows, searchEntity);
    }
    //删除
    this.dele=function(ids){
        return $http.get('../orders/delete.do?ids='+ids);
    }
    //查询一个
    this.findOne=function(id){
        return $http.get("../orders/findOne.do?id="+id);
    }
    this.update=function(entity){
        return $http.post("../orders/update.do",entity);
    }
    this.findPage = function(page,rows){
        return $http.get("../orders/findPage.do?pageNum="+page+"&pageSize="+rows);
    }

});

File: pinyougou-interface/src/main/java/cn/itcast/core/service/OrdersService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.good.Brand;
import cn.itcast.core.pojo.order.Order;
import entity.PageResult;
import vo.OrderVo;

import java.util.List;

public interface OrdersService {

    List<OrderVo> findAll(String name);

    void delete(Long[] ids);

    Order findOne(Long id);

    void update(Order order);

}
